tools/lib/provision.py

```diff
@@ -130,6 +130,11 @@
     "netcat",               # Used for flushing memcached
     "libfontconfig1",       # Required by phantomjs
     "default-jre-headless",  # Required by vnu-jar
+    "libxml2-dev",          # This bunch of packages is needed for python-xmlsec
```

Comment:
Let's put this comment on its own line above the block; I think inline comments at most makes sense for single items, and actually it'd probably be better to just move the comments to all live on their own lines for consistency.

Reply:
Moved to setup_venv.py in the first commit, since that's the correct approach after all